src/CmfContentTypeBundle.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\ContentType;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Sylius\Component\Registry\DependencyInjection\Compiler\AliasedServicePass;
```

Comment:
Using the Sylius registry component for the registry classes + the compiler pass extensions ([PR pending](https://github.com/Sylius/Sylius/pull/5485))